src/vtkIECTransformLogic.h

```diff
@@ -157,7 +157,27 @@ class VTK_IEC_TRANSFORM_LOGIC_EXPORT vtkIECTransformLogic : public vtkObject
   {
     return this->IECTransforms;
   }
+  
+  static inline  uint64_t VectorizedToLinearizedImageIndex(uint16_t i, uint16_t j , uint16_t k, uint16_t numberOfImageColumns, uint16_t numberOfImageRows, uint16_t numberOfImageSlices)
```

Comment:
I suggest to simplify the argnames as nI, nJ, nK

And add on top the Doxygen documentation saying
\param nI number of elements in first axis (columns in a DICOM image)
...
Add also that this is for row-major order